commands/util/binary.js

```diff
@@ -0,0 +1,70 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("binary")
+    .setDescription("A command with binary conversion and addition functionality")
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("convert")
+        .setDescription("Convert a number to binary")
+        .addIntegerOption(option =>
+          option
+            .setName("number")
+            .setDescription("Pick a number to convert into binary")
+            .setRequired(true)
+        )
+    )
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("add")
+        .setDescription("Add two numbers")
```

Comment:
Add finds the sum of two binary numbers. Not regular numbers

Also see if you can make it so that the user can add a regular number to a binary number. They can input either format in either input box

Reply:
no that isnt possible 10 =/= 2